cfme/tests/cloud_infra_common/test_retirement.py

```diff
@@ -189,8 +189,7 @@ def test_retirement_now_ec2_instance_backed(retire_ec2_s3_vm, tagged):
 
 @test_requirements.retirement
 @pytest.mark.tier(1)
-@pytest.mark.meta(blockers=[BZ(1419150, forced_streams='5.6',
-                               unblock=lambda: current_version() >= '5.7')])
+@pytest.mark.meta(blockers=[BZ(1516953, forced_streams='5.9')])
```

Comment:
**required** is this a general blocker or provider specific?

Reply:
The button is not present in CFME UI so I expect this influences all the providers

Reply:
thanks 👍

Reply:
dtto here.